apps/schematic/api/schematic_api/test/test_manifest_generation_controller_impl.py

```diff
@@ -45,92 +46,7 @@ def test_error_statuses(self, test_schema_url: str) -> None:
                     manifest_title="title",
                     use_strict_validation=True,
                     generate_all_manifests=False,
+                    display_label_type="class_label",
                 )
                 assert status == 404
                 assert isinstance(result, BasicError)
-
```

Comment:
Why were these removed?

Reply:
I'm not sure why those are showing up. Everythgin to do with teh google sheets endpoint is from a previous PR. I don't think the github files changed viewer is handling the 

"@andrewelamb
[Merge branch 'main' into fetaure-FDS-829-component-requirements](https://github.com/Sage-Bionetworks/sage-monorepo/pull/2667/commits/8693f40c5276cd91777fc5f4efc59a15cb81f9f7)"

correctly.